lib/embeddable_content/version.rb

```diff
@@ -1,3 +1,3 @@
 module EmbeddableContent
-  VERSION = '0.1.14'.freeze
+  VERSION = '0.1.15'.freeze
```

Comment:
is this the same version number @dylandechant is using? could be problematic (?)

Reply:
@econnally i'm flexible :)